install_config/aggregate_logging.adoc

```diff
@@ -855,7 +855,18 @@ spec:
 ----
 <1> This label should uniquely identify a replica with a single node that bears that
 label, in this case `logging-es-node=1`. Use the `oc label` command to apply
-labels to nodes as needed.
+labels to as many nodes as needed.
+
+For example, if your deployment has three infrastructures, you could add labels for those
+infrastructures as follows:
+----
+oc label node <nodename1> logging-es-node=1
+oc label node <nodename2> logging-es-node=2
+oc label node <nodename3> logging-es-node=3
+----
+
+For information about adding a label to an infrastracture, see
+https://docs.openshift.com/container-platform/3.10/admin_guide/manage_nodes.html#updating-labels-on-nodes.
```

Comment:
@jhoyt-rh should this be an xref to the manage_nodes.adoc file and to the anchor within it?

Reply:
@vikram-redhat  I'll find out how to xref to the anchor and update it.